ironfish/src/wallet/errors.ts

```diff
@@ -2,13 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+import { CurrencyUtils } from '../utils'
+
 export class NotEnoughFundsError extends Error {
   name = this.constructor.name
 
   constructor(assetId: Buffer, amount: bigint, amountNeeded: bigint) {
     super()
-    this.message = `Insufficient funds: Needed ${amountNeeded.toString()} but have ${amount.toString()} for asset '${assetId.toString(
-      'hex',
-    )}'`
+    this.message = `Insufficient funds: Needed ${CurrencyUtils.renderIron(
+      amountNeeded.toString(),
```

Comment:
`CurrencyUtils.renderIron()` takes either an encoded currency value, or just a bigint currency value so we don't need to call toString() on this.

Also, should we display the ticker here too, or no?

Reply:
removed `toString()`

Reply:
I'm torn, `needed 1 but had 0 IRON` (or whatever other ticker) seems odd but `needed 1 IRON but had 0 IRON` seems redundant

wait I'll change it to `needed 1 IRON but had 0`